corehq/motech/utils.py

```diff
@@ -98,3 +99,9 @@ def pformat_json(data):
         return json.dumps(json_data, indent=2, sort_keys=True)
     except ValueError:
         return data
+
+
+def get_commcare_users_by_location(domain_name, location_id):
```

Comment:
can you use a pattern similar to this so you don't have to pull all web users too?: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/dbaccessors.py#L69-L71 ?

Reply:
Thanks. That's much better.